dashboard/src/containers/DeploymentItemContainer/DeploymentItemContainer.ts

```diff
@@ -0,0 +1,44 @@
+import * as _ from "lodash";
+import { connect } from "react-redux";
+import { Action } from "redux";
+import { ThunkDispatch } from "redux-thunk";
+
+import actions from "../../actions";
+import DeploymentItem from "../../components/AppView/DeploymentsTable/DeploymentItem";
+import ResourceRef from "../../shared/ResourceRef";
+import { IStoreState } from "../../shared/types";
+
+interface IDeploymentItemContainerProps {
+  deployRef: ResourceRef;
+}
+
+function mapStateToProps({ kube }: IStoreState, props: IDeploymentItemContainerProps) {
+  const { deployRef } = props;
+  return {
+    name: deployRef.name,
+    deployment: kube.items[deployRef.getResourceURL()],
+  };
+}
+
+function mapDispatchToProps(
+  dispatch: ThunkDispatch<IStoreState, null, Action>,
+  props: IDeploymentItemContainerProps,
+) {
+  const { deployRef } = props;
+  return {
+    getDeployment: () =>
+      dispatch(
+        actions.kube.getResource(
+          deployRef.apiVersion,
+          "deployments",
```

Comment:
rather than hardcoding this, we should use the `deployRef.resourcePlural()` and make that a public method - I'll make this change in a future PR, once the WebSockets have moved to redux as it applies to them too.

Reply:
why don't you add it now? It seems trivial since you already have `ResourceKindToPlural` but I may be missing something.

Reply:
I don't want to change it for all of the containers in this PR, would rather do that in a separate smaller PR once this is landed.